pkg/gatherers/clusterconfig/tsdb_status.go

```diff
@@ -0,0 +1,44 @@
+package clusterconfig
+
+import (
+	"context"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+// GatherTSDBStatus gathers the Prometheus TSDB status.
+//
+// * Location in archive: config/tsdb/
+// * See: docs/insights-archive-sample/config/metrics
+// * Id in config: tsdb
```

Comment:
The id refers to https://github.com/openshift/insights-operator/pull/528/files#diff-d7e13c2682288c56fd24ed9b319c54e9eb137c510ffcb44ca035e78285ede7bcR94 so it should tsdb_status here.

Please run make docs after these minor changes and include the updated gathered-data.md here. Thank you.

Reply:
Done